README.md

```diff
@@ -12,6 +17,23 @@ Alpha AMBER API
 * On macOS: Xcode (or xcode-select), see [Nokogiri docs](http://www.nokogiri.org/tutorials/installing_nokogiri.html#mac_os_x) - `xcode-select --install`
 
 ## Installation
+### With Docker
+1. Build the project using `docker-compose -f docker-compose.development.yml build api`. This will install the dependencies and set up the image. If dependencies are updated/added, you need to run this command again.
+2. Copy the `.env.example` to `.env` and update the fields to reflect your environment. To allow the development Docker configuration on amber-ui to work, change `COMPOSE_PROJECT_NAME` to "amber_development".
+3. Run the project using `docker-compose -f docker-compose.development.yml up api`.
+4. Create databases and tables and run seeds with `bundle exec rails db:setup` (see tip on how to run commands in containers).
+
+Tip: to run commands in a container, you need to find the container's ID using the `docker ps` command first. Copy the ID of the container with image "amber_development_api".
```

Comment:
You can also do `docker-compose exec <service name> <command>` when the service is running

Reply:
The service name is appended with a _1 and possibly higher number if somehow more containers of the same name are running, hence I thought the CONTAINER_ID would be clearer here.
Shall I change it or is it ok this way?